package.json

```diff
@@ -32,5 +32,8 @@
   "scripts": {
     "boosh": "smoosh make ./build.json",
     "test": "node ./test.js"
+  },
+  "dependencies": {
+    "xhr2": "0.1.0"
```

Comment:
It worked for me in Webpack.  I've since moved on to https://github.com/matthew-andrews/isomorphic-fetch

Feel free to make any changes you see fit to make it more generic.